modules/govuk_jenkins/manifests/job/govuk_tagging_monitor.pp

```diff
@@ -0,0 +1,23 @@
+# == Class: govuk_jenkins::job::govuk_tagging_monitor
+#
+# Monitor GOV.UK navigation pages to find invalid pages that we shouldn't
+# display to users. It also raises alerts when navigation pages exceed the
+# number of allowed links, so a content designer can act on those. The ruby
+# script handles slack notifications.
+#
+# === Parameters
```

Comment:
It is a bit tricky because the slack plugin in jenkins is configured with the token for the 2nd line slack channel and not ours. I think it's fine to leave it for now if you agree.